azure.yaml

```diff
@@ -0,0 +1,37 @@
+# yaml-language-server: $schema=https://raw.githubusercontent.com/Azure/azure-dev/main/schemas/v1.0/azure.yaml.json
+
+name: azure-search-openai-demo
```

Comment:
Should 'name' and 'template' be different, e.g. aisearch-openai-rag-audio? Not sure if this was copy/paste from other sample.

Reply:
Yes, thank you! Copypasta